### PR TITLE
wish list-items added

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**WISHLIST MANAGEMENT Page**

- Added cards 
- cross button to remove item from wish list
WishList (1)

### CODE REVIEW

**Review by VedanthB — APPROVED** [review]
looks good to go 🚀

File: pages/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  flex-wrap: wrap;
+
+  padding: 0rem 2rem;
+  justify-content: space-between;
+}
+
+.product-img-div {
+  position: relative;
+}
+.fa-xmark {
```
Comment (pritam-kr) [inline pages/wishlist.css:12]: You can write a more meaningful class for this.

File: pages/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  color: #9ca3af;
```
Comment (pritam-kr) [inline pages/wishlist.css:13]: Use variable for colors

**Review by pritam-kr — APPROVED** [review]
- Design look clean. That's a good thing
- Look good to me

File: pages/wishlist.css (hunk 1 above)
Comment (tanishk15sharma) [reply pages/wishlist.css:12]: yes

File: pages/wishlist.css (hunk 2 above)
Comment (tanishk15sharma) [reply pages/wishlist.css:13]: yaa I am working on it

**Pull request merged**